### PR TITLE
WIP: Started on lecture 1 slides

### PR DESCRIPTION
**Pull request by mauro3**
Compile to notebooks with `make_notebooks.jl`.

Does this look about right?

### CODE REVIEW

File: slide-notebooks/make_notebooks.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#     end
+
+#     return str
+# end
+
+for fl in readdir()
+    if splitext(fl)[end]!=".jl" || splitpath(@__FILE__)[end]==fl
+        continue
+    end
+    Literate.notebook(fl, "notebooks", credit=false, execute=false) #, preprocess=include2nbinclude)
```
Comment (luraess) [inline slide-notebooks/make_notebooks.jl:35]: won't the `make_notebook.jl` script be included as well ?

Comment (luraess) [reply slide-notebooks/make_notebooks.jl:35]: Also, maybe add a copy function to move the `.jl` scripts to `website/_literate`?

File: exercises-notebook/make_notebooks.jl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#     end
+
+#     return str
+# end
+
+for fl in readdir()
+    if splitext(fl)[end]!=".jl" || splitpath(@__FILE__)[end]==fl
+        continue
+    end
+    Literate.notebook(fl, "notebooks", credit=false, execute=false) #, preprocess=include2nbinclude)
```
Comment (luraess) [inline exercises-notebook/make_notebooks.jl:35]: Same comments here as for the `slide-notebook/make_notebooks.jl` file.

**Comment by luraess** [comment]
Also, as from the current status, the `README.md` and `website-memo.md` files are ahead of this branch.

File: slide-notebooks/make_notebooks.jl (hunk 1 above)
Comment (mauro3) [reply slide-notebooks/make_notebooks.jl:35]: `splitpath(@__FILE__)[end]==fl` should check whether `fl` is the same as the current file.

Comment (luraess) [reply slide-notebooks/make_notebooks.jl]: Got it.

File: website/software_install.md [hunk 3]
```diff
@@ -9,8 +9,17 @@ hascode = true
 
 ## Opening and running the Jupyter Julia notebook
```
Comment (luraess) [inline website/software_install.md:10]: @mauro3 Would be good to add infos here on how students can deal with notebooks, in particular for the homework. One should describe workflow on how to open the notebook, save a personal copy, edit it, then save the folder containing the notebooks to then upload it to Moodle.

**Pull request merged**